DPG reference catalog for open standards

```diff
@@ -0,0 +1,54 @@
+###DPG Reference catalog for open standards
+##This reference catalog is an overview of standards that are recommended for the Digital Public Goods. The standards can be technical or semantic, and are sorted by relevant application. 
+
+#Datasets:
+JSON (JavaScript Object Notation)
+XML
+
+#Character encoding and languages:
+UTF-8, ISO/IEC 10646
+ISO 639-1/639-2/639-3
+
+#Web standards:
+HTML 
+CSS
+SSL
+HTTP
+XHTML (eXtensible HTML v1.0)
+FTP
```

Comment:
The use of FTP (vs. more secure protocols like HTTPS or SFTP) is generally discouraged in technical communities due to its age and security status, and it is noteworthy that its support was removed from most browsers in 2020. More info: https://groups.google.com/g/mozilla.dev.platform/c/FqCZUT9ay_o/m/jt4DLRDjAwAJ

Reply:
Agreed. FYI @nathanbaleeta